mlflow/models/output_parsers.py

```diff
@@ -0,0 +1,60 @@
+from dataclasses import asdict
```

Comment:
Can we add this to the MLflow documentation? https://github.com/mlflow/mlflow/blob/master/docs/source/python_api/mlflow.models.rst